lib/pbench/server/indexing_tarballs.py

```diff
@@ -267,6 +272,32 @@ def process_tb(self, tarballs):
         else:
             idxctx.set_tracking_id(tracking_id)
 
+        return res.value
+
+    def process_tb(self, tarballs: List[TarballData]) -> int:
+        """Process Tarballs For Indexing and creates report
+
+        Args:
+            tarballs:   List of tarball information tuples
+
+        Returns:
+            status code
+        """
+        res = 0
+        idxctx = self.idxctx
+        error_code = self.error_code
+
+        # Store the list of tarballs as a deque
+        tb_deque = deque(tarballs)
+
+        res = self.load_templates()
+        if res:
+            idxctx.logger.info("Load templates {!r}", res)
+            return res
+
+        # tarballs contains a list of tar balls sorted by size.
+        idxctx.logger.debug("Preparing to index {:d} tar balls", len(tb_deque))
```

Comment:
The factoid in this comment is, once again, fascinating, but it doesn't seem particularly germane to the code around it.  (In particular, I think we are done with `tarballs` and we're working with `tb_deque`, now.)  So, I would just omit this comment.